packages/utils-dom/src/htmlelement.ts

```diff
@@ -260,6 +260,13 @@ export class HTMLElement extends Node {
         return parse5.serialize(this._element as htmlparser2Adapter.Node, { treeAdapter: htmlparser2Adapter });
     }
 
+    /**
+     * https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelectorAll
+     */
+    public querySelectorAll(selectors: string): HTMLElement[] {
+        return this.ownerDocument.querySelectorAll(selectors);
```

Comment:
We need to scope this query to only include results representing nodes under the element it was called on. To do that, I suspect we'll need to call `cssSelect` directly so we can specify the proper scope.

Reply:
good catch, done.